tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -92,6 +94,47 @@ where
 		Ok(json!(json_value.to_hex()))
 	});
 
+	// author_getNextNonce
+	let author_get_next_nonce: &str = "author_getNextNonce";
+	io.add_sync_method(author_get_next_nonce, move |params: Params| {
+		debug!("author_getNextNonce in enclave");
+		match params.parse::<(String, String)>() {
+			Ok((shard_base58, account_hex)) => {
+				let shard = match decode_shard_from_base58(shard_base58.as_str()) {
+					Ok(id) => id,
+					Err(msg) => {
+						let error_msg: String =
+							format!("Could not retrieve pending trusted calls due to: {}", msg);
```

Comment:
I think the message is misleading. The RPC call is about  getting next nonce, not pending trusted calls.